newrelic/common/system_info.py

```diff
@@ -74,7 +75,7 @@ def logical_processor_count():
     # looking at the devices corresponding to the available CPUs.
 
     try:
-        pseudoDevices = os.listdir("/devices/pseudo/")
+        pseudoDevices = Path("/devices/pseudo/").iterdir()
```

Comment:
I don't think these locations make sense on windows but I assume we'll address that later.

Reply:
No but according to the commend above in this file, that line is not reachable as Windows is already handled. That block is only for checking on Solaris systems I think based on the other comment.


```
    # The multiprocessing module provides support for Windows,
    # BSD systems (including MacOS X) and systems which support
    # the POSIX API for querying the number of CPUs.

    try:
        return multiprocessing.cpu_count()
    except NotImplementedError:
        pass
```

```
    # Assuming that Solaris will support POSIX API for querying
    # the number of CPUs. Just in case though, work it out by
    # looking at the devices corresponding to the available CPUs.

    try:
        pseudoDevices = os.listdir("/devices/pseudo/")
```